src/rpc/mining.cpp

```diff
@@ -97,7 +97,8 @@ void ParsePrivateKeyList(const UniValue& privkeys_hex, std::vector<CKey>& vecKey
     for(int i=0; i< privkeys_hex.size(); i++) {
         const UniValue& privkey = privkeys_hex[i];
         std::string keyHex = privkey.get_str();
-        if(keyHex.length() % 64 != 0) {
+        if(keyHex.length() != 64 
+        && keyHex.length() != 52) {
```

Comment:
What is the length 52 check for?

Reply:
this change should not be here, sorry. it was made while testing the issue reported on error message.